src/Command/CreateFamilyCommand.php

```diff
@@ -120,6 +120,16 @@ public function execute(InputInterface $input, OutputInterface $output)
             $categoryOptions = array_filter(explode(',', $categoryOptions), function (string $categoryOption) {
                 return !empty($categoryOption);
             });
+            $numberOfOptions = \count($categoryOptions);
+            if ($numberOfOptions > 100) {
+                $this->io->warning(
```

Comment:
if it fail, it's an error.

Reply:
Yes, when the user sets the option to convert the categories to multiselect but at the same time has more than 100, it fails by explaning him why.

It does not really change the behavior does it ?